core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -397,6 +397,12 @@ public String toString() {
                           .toString();
     }
 
+    public void reconfigure(ReconfigurableServer rs) {
+        final ServerBuilder sb = rs.reconfigure(Server.builder());
+        // Here we build our new configuration object. Now time to replace the old config with the new.
+        final ServerConfig config = sb.buildServerConfig(this.config());
```

Comment:
Here I have built the new ServerConfig. The serverConfig in Server class is final so really cannot replace it. Are we ok to change that ?

Reply:
Yeah, I think so.